catroid/src/main/java/org/catrobat/catroid/stage/StageListener.java

```diff
@@ -215,6 +218,11 @@ public void create() {
 				stage.addActor(penActor);
 				addPenActor = false;
 			}
+			if (addEmbroideryActor) {
```

Comment:
if i get this right, this is meant to add the embroideryActor to the very first sprite only?
can you please do that explicitly outside the loop. (same with the penActor while at it, and same in render() aswell)